example/ck_tile/40_streamk_gemm/run_gemm_example.inc

```diff
@@ -271,9 +271,10 @@ int run_gemm_example_with_layouts(int argc,
     std::cout << "Run Gemm kernel with M=" << M << " N=" << N << " K=" << K
               << " StrideA=" << stride_A << " StrideB=" << stride_B << " StrideC=" << stride_C
               << " A_Layout=" << ALayout::name << " B_Layout=" << BLayout::name
-              << " C_Layout=" << CLayout::name << " A_Type=" << DataTypeTraits<ADataType>::name
-              << " B_Type=" << DataTypeTraits<BDataType>::name
-              << " C_Type=" << DataTypeTraits<CDataType>::name
+              << " C_Layout=" << CLayout::name
```

Comment:
At the top of this file: Should add #include "ck_tile/ops/common/utils.hpp" . Don't want to rely on someone else including it.